tests/Makefile

```diff
@@ -0,0 +1,7 @@
+test: test-integration test-e2e 
```

Comment:
can we import this in the main makefile to expose the commands?

Reply:
Sure added!